tutor/commands/compose.py

```diff
@@ -7,12 +7,10 @@
 from tutor import bindmounts
 from tutor import config as tutor_config
 from tutor import env as tutor_env
-from tutor import fmt, jobs, utils
-from tutor import serialize
+from tutor import fmt, hooks, jobs, serialize, utils
```

Comment:
Import order changes are the result of running `make isort` and you can safely ignore them.